recipes/_install.rb

```diff
@@ -84,7 +84,7 @@ def nsp
 windows_password = CernerSplunk::DataBag.load(node['splunk']['windows_password'], secret: node['splunk']['data_bag_secret'])
 
 execute 'splunk-first-run' do
-  command "#{node['splunk']['cmd']} help commands --accept-license --answer-yes --no-prompt"
+  command "#{node['splunk']['cmd']}  start  --accept-license --answer-yes --no-prompt --seed-passwd '#{node['splunk']['package']['seed']['passwd']}' "
```

Comment:
What happens if you run this on a pre-7.2.x version of Splunk?  We should maintain backward compatibility to at least 7.0.  If the --seed-passwd is just ignored then great, otherwise we may need to flex the command based on what version of Splunk is being installed.

Reply:
updated the _install.rb file to support pre-7.2 and post 7.2